js/src/WalletLink.ts

```diff
@@ -2,14 +2,14 @@
 // Copyright (c) 2018-2020 Coinbase, Inc. <https://www.coinbase.com/>
 // Licensed under the Apache License, version 2.0
 
-import { WalletLinkProvider } from "./provider/WalletLinkProvider"
```

Comment:
Bitcoin

Reply:
Yeah I'm lost I don't know how to do all that